### DISCUSSION TITLE
Cursor on Listbox

### DISCUSSION THREAD

**Discussion by zakiego** [shuding/nextra#496] [Ideas]
**Preview** https://nextra-kappa.vercel.app

How about changing the `cursor-default` on the `Listbox` to `cursor-pointer`?

Because on light mode, it's hard to know that it's a menu

https://github.com/shuding/nextra/blob/e162f839a4967ce66345a57e9b43c30651fffb93/packages/nextra-theme-docs/src/select.tsx#L22-L29
---

Example with `cursor-default`

https://user-images.githubusercontent.com/78015359/175844363-3820fca1-3ca3-47ee-a151-d9dd7ca5f63e.mp4

---

Example with `cursor-pointer`

https://user-images.githubusercontent.com/78015359/175844370-13357c19-15a7-4987-8785-3465487ed8e7.mp4

**Comment by dimaMachina** [comment]
Nextra 3 now have cursor
